plugin/http_client.py

```diff
@@ -12,14 +12,15 @@
 if not from_cmdline:
     import vim
 
-
 METHOD_REGEX = re.compile('^(GET|POST|DELETE|PUT|HEAD|OPTIONS|PATCH) (.*)$')
 HEADER_REGEX = re.compile('^([^()<>@,;:\<>/\[\]?={}]+):\\s*(.*)$')
 VAR_REGEX = re.compile('^# ?(:[^: ]+)\\s*=\\s*(.+)$')
 GLOBAL_VAR_REGEX = re.compile('^# ?(\$[^$ ]+)\\s*=\\s*(.+)$')
 FILE_REGEX = re.compile("!((?:file)|(?:(?:content)))\((.+)\)")
 JSON_REGEX = re.compile("(javascript|json)$", re.IGNORECASE)
 
+verify_ssl = True if vim.eval('g:http_client_verify_ssl') == '1' else False
```

Comment:
Maybe: `verify_ssl = vim.eval('g:http_client_verify_ssl') == '1'` for simplicity?